package.json

```diff
@@ -54,12 +60,12 @@
   "types": "src/index.d.ts",
   "scripts": {
     "build": "npm-run-all --parallel build:* && cp dist/preact.js dist/preact.min.js",
-    "build:core": "microbundle build --raw",
-    "build:debug": "microbundle build --raw --cwd debug",
-    "build:devtools": "microbundle build --raw --cwd devtools",
-    "build:hooks": "microbundle build --raw --cwd hooks",
-    "build:test-utils": "microbundle build --raw --cwd test-utils",
-    "build:compat": "microbundle build --raw --cwd compat --globals 'preact/hooks=preactHooks'",
+    "build:core": "microbundle build --raw && cp ./dist/preact.module.js ./dist/preact.mjs",
```

Comment:
Might want to convert these to `npm-run-all` like "build" is so they work on Windows based systems.

Reply:
[&& works on Windows](https://ss64.com/nt/syntax-redirection.html). `cp` does not. Might need to use https://npm.im/cpy-cli

Reply:
+1 for cpy